docker-compose.yml

```diff
@@ -179,6 +227,53 @@ services:
       - fence-service
       - portal-service
       - pidgin-service
+  tube-service:
```

Comment:
For ETL we have a separate `docker-compose.yaml` with Elasticsearch also, can you update that one?